samples/bidirectional-chat/README.md

```diff
@@ -0,0 +1,115 @@
+# Azure function bidirectional chatroom sample
+
+This is a chatroom sample that demonstrates bidirectional message pushing between Azure SignalR Service and Azure Function in serverless scenario. It leverage the "upstream" provided by Azure SignalR Service that features proxying messages from client to upstream endpoints in serverless scenario. Azure Functions with signalr trigger binding allow you to write code to receive and push messages in several languages, including JavaScript, Python, C#, etc.
```

Comment:
>signalr [](start = 319, length = 7)

SignalR

Reply:
Azure Functions is plural so it should be `allow`?

Reply:
yeah, then update SignalR is enough.

---
In reply to: [422756824](https://github.com/Azure/azure-functions-signalrservice-extension/pull/120#discussion_r422756824) [](ancestors = 422756824)